src/constants.ts

```diff
@@ -0,0 +1,271 @@
+export const careSetting = "6f0c9a92-6f24-11e3-af88-005056821db0";
+
```

Comment:
Are those concepts necessary? where are they used

Reply:
This was deleted as well as it was not being used anywhere.